src/main/resources/theme/rciam/login/login-oauth-grant.ftl

```diff
@@ -13,16 +13,18 @@
         </p>
     <#elseif section = "form">
         <div id="kc-oauth" class="content-area">
-            <h3>${msg("oauthGrantRequest")}</h3>
-            <ul>
-                <#if oauth.clientScopesRequested??>
```

Comment:
I actually moved this #if one level up, to include the message...
I don't know if you agree that if the list is empty, it's better not to show a message 
"This service would like to have acccess to:"

...with nothing below...

Reply:
This is the situation for a client with consent required, Display Client On Consent Screen off and without scopes with consent.
I am not sure what is best for such a situation. Maybe another message?

Reply:
Indeed, displaying a different message if empty list, would be the best!

Reply:
ok, let's change that